services/requester/requester.go

```diff
@@ -406,6 +412,24 @@ func (e *EVM) GetCode(
 	return code, nil
 }
 
+func (e *EVM) GetLatestEVMHeight(ctx context.Context) (uint64, error) {
```

Comment:
add a todo to "think about adding some time-based caching" since this might be very frequent request, but again until we have metrics showing that we shouldn't, just good to have a todo as a reminder to think about.

Reply:
Good point :+1: 
Added in https://github.com/onflow/flow-evm-gateway/pull/180/commits/59699f006b32fc2b59c77c34da7f4c2d9d287b50